Chatkit/Dependencies/Dependencies.swift

```diff
@@ -86,4 +87,9 @@ class ConcreteDependencies: Dependencies {
     var store: Store {
         return dependencyFactory.resolve(Store.self, dependencies: self)
     }
+    
+    var reductionManager: ReductionManager {
+        return dependencyFactory.resolve(ReductionManager.self, dependencies: self)
```

Comment:
You need to update the tests in`ConcreteDependencyTests` to account for this new property.  And I think those tests might fail because you've not `registered` the `ConcreteReductionManager` in the initialiser. 

Also whilst you're in `ConcreteDependencyTests` would you mind correcting some mistakes I've made 🙏🏼...
- the file/class should be name `ConcreteDependenciesTests` 
- it should live in the `Tests/Dependencies` directory/group but its currently in `Tests/Redux/Store`

Reply:
Sure, I will have a look into that.

Reply:
Done